ext/rbs_extension/main.c

```diff
@@ -272,6 +272,92 @@ static VALUE rbsparser_parse_signature(VALUE self, VALUE buffer, VALUE start_pos
   return result;
 }
 
+static VALUE parse_inline_leading_annotation_try(VALUE a) {
+  struct parse_type_arg *arg = (struct parse_type_arg *)a;
```

Comment:
We've been putting spaces after casts

```suggestion
  struct parse_type_arg *arg = (struct parse_type_arg *) a;
```

Reply:
Yeah we can add clang-format, like Sorbet uses.

We have nothing against, we've just been focusing on more important stuff so far

Reply:
So I ended up spending quite some time setting it up: https://github.com/ruby/rbs/pull/2437
I think it's better to upstream the formatting setup first then having it as part of a huge PR.